index.html

```diff
@@ -22,36 +22,47 @@
           id="headeline-bg"
         />
         <header id="page-head">
-          <div id="header-logo">
-            <a href="#" id="logo">Welcome</a>
-          </div>
           <nav id="navbar">
-            <div id="nav-links">
-              <a href="#" class="nav-link"
-                ><i class="fa fa-bars" aria-hidden="true"></i
-              ></a>
-              <ul id="menu-items">
-                <li>
-                  <a href="#" class="nav-link">Portfolio</a>
-                </li>
-                <li>
-                  <a href="#" class="nav-link">About</a>
-                </li>
-                <li>
-                  <a href="#" class="nav-link">Contact</a>
-                </li>
-              </ul>
+            <div id="nav-top">
+              <div id="header-logo">
+                <a href="#" id="logo">Welcome</a>
+              </div>
+              <div id="nav-links">
+                <a href="#" class="nav-link"
+                  ><i class="fa fa-bars" aria-hidden="true"></i
+                ></a>
+              </div>
             </div>
+            <div class="menu">
+            <ul id="menu-items">
+              <li id="close-menu">
+                <p id="close"><i class="fa fa-times" aria-hidden="true"></i></p>
+              </li>
+              <li>
+                <a href="#" class="nav-link">Portfolio</a>
+              </li>
+              <li>
+                <a href="#" class="nav-link">About</a>
+              </li>
+              <li>
+                <a href="#" class="nav-link">Contact</a>
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] Kindly make sure when the user clicks on a nav link in the mobile menu or the desktop view, the page is scrolled to the appropriate section of the page. You can use the `href` property.

- [ ] Please make sure that the mobile menu is displayed on the whole page and it can't scrolled to the main page's content. (Check the below gif)

![back-end GIF](https://user-images.githubusercontent.com/62290545/167643041-82d3044c-0a27-45b9-83e4-297ee6e41ba3.gif)